<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>931</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFunction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>F(x) = -|x|³ + 3x²</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F(x) = sin(x)/(1+x²)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F(x) = sin(x)/(1+x²) + cos(tx)</string>
     </property>
    </item>
   </widget>
   <widget class="QTableView" name="table1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>461</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QTableView" name="table2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>791</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="buttonSolve">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Решить!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditT">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditN">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelN">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>270</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>n:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelT">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>300</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>t:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSa">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>210</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>S''(a):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSa">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSb">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>S''(b):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSb">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelB">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>178</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>b:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>178</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelA">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>148</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>148</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxF">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>F(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxS">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>S(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxD">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>F(x) - S(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxD1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>F'(x) - S'(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxF1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>F'(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxS1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>S'(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxF2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>F''(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxS2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>S''(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxD2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>F''(x) - S''(x)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelShowGraph">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Показать/скрыть&lt;br/&gt;графики:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выберите задачу и&lt;br/&gt;введите входные данные:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSolnInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
